solr/conf/schema.xml

```diff
@@ -194,14 +202,29 @@
     <dynamicField name="*_llsi" type="location" stored="true" indexed="true" multiValued="false"/>
     <dynamicField name="*_llsim" type="location" stored="true" indexed="true" multiValued="true"/>
 
-    <dynamicField name="*_srpt" type="location_rpt" stored="true" indexed="true" multiValued="true" />
+    <dynamicField name="*_srpt" type="location_rpt" stored="true" indexed="true" multiValued="true"/>
 
     <!-- suggest and spelling -->
-    <dynamicField name="*spell" type="textSpell" indexed="true" stored="false" multiValued="true" />
-    <dynamicField name="*suggest" type="textSuggest" indexed="true" stored="false" multiValued="true" />
+    <dynamicField name="*spell" type="textSpell" indexed="true" stored="false" multiValued="true"/>
+    <dynamicField name="*suggest" type="textSuggest" indexed="true" stored="false" multiValued="true"/>
 
     <!-- JSON -->
-    <dynamicField name="*_struct" type="string" stored="true" indexed="false" multiValued="true" omitNorms="true" />
+    <dynamicField name="*_struct" type="string" stored="true" indexed="false" multiValued="true" omitNorms="true"/>
+
+    <!-- docValue tests -->
+    <!-- Facets -->
+    <field name="access_facet"        type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="all_authors_facet"   type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="format"              type="string" docValues="true" indexed="false" stored="false" multiValued="true"/>
+    <field name="genre_facet"         type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="genre_full_facet"    type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="language_facet"      type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="lc_1letter_facet"    type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="pub_date_facet"      type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+    <field name="subject_topic_facet" type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
+
+    <!-- Sorts -->
+    <field name="title_sort"          type="string" docValues="true" indexed="false" stored="false" multiValued="true" />
```

Comment:
@cdmo solr doc says for sorting on a field, multivalued should be false: https://lucene.apache.org/solr/guide/6_6/common-query-parameters.html#CommonQueryParameters-ThesortParameter 🤔

Reply:
> If the field is multi-valued, Lucene will use the SORTED_SET type.

from [solr docs](https://lucene.apache.org/solr/guide/7_4/docvalues.html#enabling-docvalues)

So, it appears to be legal - but how it works in practice... not sure.

Reply:
cool! thanks for the info, I just did not know :)

Reply:
I don't know either Banu! We're working through it :).

Just to document our talk, we're going to leave this as is for the test run and then do a follow up test run where we change title sorting back to the way it was. 
Thanks!